components/raftstore/src/store/config.rs

```diff
@@ -197,6 +197,15 @@ pub struct Config {
     // It will be set to raft_store_max_leader_lease/4 by default.
     pub renew_leader_lease_advance_duration: ReadableDuration,
 
+    // Set true to allow handling request vote messages within one election time
+    // after TiKV start.
+    //
+    // Note: set to true may break leader lease. It should only be true in tests.
+    #[doc(hidden)]
+    #[serde(skip)]
+    #[online_config(skip)]
+    pub allow_unsafe_vote_after_start: bool,
```

Comment:
why do we need a config?

Reply:
To speed up tests, it's allowed in testing.

Reply:
Note that this config will not be serialized or deserialized so it won't cause any configuration compatibility issues.